packages/kit/src/runtime/client/client.js

```diff
@@ -1548,7 +1548,7 @@ export function create_client({ target }) {
 					redirect_chain: [],
 					details: {
 						state: {},
-						replaceState: url.href === location.href
+						replaceState: options.replace_state || url.href === location.href
```

Comment:
shouldn't we use `??` here instead of `||` so that it's definitely set to false if we use `data-sveltekit-replacestate="off"`?